Jenkinsfile

```diff
@@ -7,6 +7,18 @@ private boolean lastCommitIsBumpCommit() {
     }
 }
 
+node {
+    checkout([
+        $class: 'GitSCM',
+        branches: scm.branches,
+        doGenerateSubmoduleConfigurations: scm.doGenerateSubmoduleConfigurations,
+        extensions: scm.extensions + [
+            [$class: 'UserExclusion', excludedUsers: 'trueconnectivity-build']
```

Comment:
this does the skipping, I understand. Feels neater than the explicit failing build 👍 do you have to define all the variables, as they seem to be defaulted?